src/main/kotlin/org/elm/lang/core/psi/elements/ElmFunctionDeclarationLeft.kt

```diff
@@ -69,4 +64,24 @@ class ElmFunctionDeclarationLeft : ElmStubbedNamedElementImpl<ElmFunctionDeclara
             val p = parent
             return p is ElmValueDeclaration && p.isTopLevel
         }
+
+    override fun getUseScope(): SearchScope {
```

Comment:
This was necessary so that we can trigger the inplace rename refactoring automatically, but it also provides a nice speed bonus for anyone who wants to rename an inner decl in a `let`.